SVS/model/utils/SVSDataset.py

```diff
@@ -128,7 +128,7 @@ def __init__(
         self.phone_size = phone_size - 1
         self.n_mels = n_mels
 
-    def __call__(self, batch):
+    def __call__(self, batch, args):
```

Comment:
you only use vocoder_category for the function, there is no need to pass the whole args to the function